fails:Hash#initialize_copy is private
fails:Hash#initialize_copy raises a RuntimeError if called on a frozen instance that is modified
fails:Hash#initialize_copy raises a RuntimeError if called on a frozen instance that would not be modified
